src/tlog-test-fd-json-reader.c

```diff
@@ -217,8 +219,8 @@ main(void)
     {.type = OP_TYPE_LOC_GET,                       \
      .data = {.loc_get = {.exp_loc = _exp_loc}}}
 
-#define TEST(_name_token, _input, _op_list_init_args...) \
-    passed = test(#_name_token,                                 \
+#define TEST(_name_token, _input, _op_list_init_args...)        \
```

Comment:
Please leave only one space before the wrapping backslash on the first line of the macro. This makes it easier to see where it starts. Like this:

    #define TEST(_name_token, _input, _op_list_init_args...) \
        passed = test(__FILE__, __LINE__, #_name_token,             \